x-pack/test/encrypted_saved_objects_api_integration/tests/encrypted_saved_objects_api.ts

```diff
@@ -437,15 +437,15 @@ export default function ({ getService }: FtrProviderContext) {
       }:${id}`;
     }
 
-    afterEach(async () => {
```

Comment:
__note:__ we need this hook just for basic tests, not for migration and key rotation tests.